### PR TITLE
fixed remaining localization strings

### PR DESCRIPTION
**Pull request by jennxu23**
Jira: [https://bugs.benetech.org/browse/CHI-701](https://bugs.benetech.org/browse/CHI-701)

In this PR, the Cases and Contacts tabs in the search results are localizable. The results are also correctly singular/plural depending on the number of results.

Screen Shot 2021-06-25 at 12 01 37 PM

### CODE REVIEW

File: plugin-hrm-form/src/components/search/SearchResults/index.tsx [hunk 1]
```diff
@@ -183,26 +186,38 @@ const SearchResults: React.FC<Props> = ({
...
+                        &nbsp;
+                        {contactsCount} <Template code="PreviousContacts-Contact" />
+                      </>
                     ) : (
-                      <Template code="PreviousContacts-Contacts" />
+                      <>
+                        <Template code="PreviousContacts-ThereAre" />
+                        &nbsp;
+                        {contactsCount} <Template code="PreviousContacts-Contacts" />
+                      </>
```
Comment (GPaoloni) [inline plugin-hrm-form/src/components/search/SearchResults/index.tsx:220]: Just highlighting this, please read below comment.

File: plugin-hrm-form/src/components/search/SearchResults/index.tsx [hunk 2]
```diff
@@ -226,7 +241,12 @@ const SearchResults: React.FC<Props> = ({
             <>
               <StyledContactResultsHeader>
                 <StyledCount data-testid="ContactsCount">
-                  {contactsCount} <Template code="SearchResultsIndex-Contacts" />
+                  {contactsCount}{' '}
+                  {contactsCount === 1 ? (
+                    <Template code="PreviousContacts-Contact" />
+                  ) : (
+                    <Template code="SearchResultsIndex-Contacts" />
+                  )}
```
Comment (GPaoloni) [inline plugin-hrm-form/src/components/search/SearchResults/index.tsx:249]: I see that there are two different approaches for formatting this with the blank spaces, one being `{' '}` and the other using `&nbsp;` character. Can we standardize and use the same? I don't have a strong preference but maybe `{' '} ` is more intuitive (as it's just a string, no special characters). Maybe it does not render accordingly tho, in that case ignore this.

**Review by GPaoloni — APPROVED** [review]
Looks good!

File: plugin-hrm-form/src/components/search/SearchResults/index.tsx (hunk 2 above)
Comment (jennxu23) [reply plugin-hrm-form/src/components/search/SearchResults/index.tsx:249]: I tried standardizing everything with `&nbsp;`, but it causes some of the checks to fail. I think we can revisit this once I figure out what's going on with the tests, but at the current state the code is functioning and passes all the checks.

**Pull request merged**